src/common/types.ts

```diff
@@ -41,20 +43,27 @@ export interface SubSerializedUser {
   profilePictureURL: string | null;
 }
 
-/* Represents a User session */
+/** Represents a User session */
 export interface SerializedUserSession {
   accessToken: string;
   active: boolean;
   refreshToken: string;
   sessionExpiration: string;
 }
 
-/* Represents an Availability */
+/** Represents an Availability */
 export type SerializedAvailability = {
   day: string;
   times: number[];
 };
 
+/** Represents a Location */
+export type SerializedLocation = {
+  area: string;
```

Comment:
what is `area` ? is it address?

Reply:
`area` is described in the entity file and it's just the general 'area' of the location (Campus, Collegetown, etc). I added some extra comments and examples to make it clearer though!